src/mgl-pax.el

```diff
@@ -407,7 +383,7 @@ See `mgl-pax-autoload'. If nil, then a free port will be used."
             (scan-error
              (goto-char end)))
           (let* ((sexp-end (point))
-                 (sexp (slime-trim-whitespace
+                 (sexp (string-trim
```

Comment:
Nice catch.

Reply:
That seems to have been introduced around Emacs 24.4. I'm considering sending a patch to SLIME making that replacement, but the manual still claims Emacs 23 compatibility so I'm not sure it would be welcome.